networkx/algorithms/tests/test_distance_measures.py

```diff
@@ -87,9 +87,25 @@ def test_radius(self):
     def test_periphery(self):
         assert set(nx.periphery(self.G)) == {1, 4, 13, 16}
 
+    def test_center_simple_tree(self):
+        G = nx.Graph([(1, 2), (1, 3), (2, 4), (2, 5)])
+        assert nx.center(G) == [1, 2]
+
+    def test_center_balanced_tree(self):
+        for r in range(2, 5):
+            for h in range(1, 5):
+                G = nx.balanced_tree(r, h)
+                assert nx.center(G) == [0]
```

Comment:
Let's go ahead and parametrize this

```suggestion
    @pytest.mark.parametrize("r", range(2, 5))
    @pytest.mark.parametrize("h", range(1, 5))
    def test_center_balanced_tree(self, r, h):
        G = nx.balanced_tree(r, h)
        assert nx.center(G) == [0]
```

Reply:
Done!